### PR TITLE
add new project attrs, update seeds

### LINKED ISSUE #29: Update data models to reflect the Teach team's requirements
Our data models need to evolve to identically map the UX workflow and representation of data on Thimble's side.

One of the recent changes that we discussed is to add a `tags` attribute to the projects model to store project-related tags that a user might specify. Another change was to add a `description` attribute to the project as well.

We probably eventually want to include thumbnails as well for projects depending on how the UX team decides to implement thumbnails.

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #29

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
lemme know what you think @gideonthomas

File: migrations/1_create_users_projects_files.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ exports.up = function (knex) {
       t.increments('id');
       t.integer('user_id').notNullable().references('id').inTable('users');
       t.text('title').notNullable();
+      t.text('tags');
```
Comment (sedge) [inline migrations/1_create_users_projects_files.js]: I guess we'll have to parse them ourselves? Maybe store as csv...

Comment (gideonthomas) [inline migrations/1_create_users_projects_files.js]: maybe we want a separate table for this? That might make things even easier when the Thimble gallery is implemented (for e.g. filtering makes by tags)?

**Comment by gideonthomas** [comment]
this looks good to me!

File: migrations/1_create_users_projects_files.js (hunk 1 above)
Comment (ashleygwilliams) [inline migrations/1_create_users_projects_files.js]: when we do this in pg it shouldn't be so bad. http://www.postgresql.org/docs/9.1/static/arrays.html

Comment (gideonthomas) [inline migrations/1_create_users_projects_files.js]: yeah but it will completely break in dev if we use sqlite

**Pull request merged**